### DISCUSSION TITLE
Can't access my registered account on the Heroku deployed site

### DISCUSSION THREAD

**Discussion by devsoftzz** [WebMetadataRetrieval/15-Web_Metadata_Retrieval#59] [Q&A] [answered]
I have registered yesterday on the website. But today I'm not able to log in with the same email id, it shows the user doesn't exist.
@harshilol @ahirthecoder Maybe it's a hosting side problem, as in the local machine the website works seemlessly.

**Comment by harshilg10** [comment, marked as answer]
This happens Due to Dyno Manager

The Heroku filesystem is ephemeral - that means that any changes to the filesystem whilst the dyno is running only last until that dyno is shut down or restarted. Each dyno boots with a clean copy of the filesystem from the most recent deploy. This is similar to how many container based systems, such as Docker, operate. To understand in depth check this link: https://devcenter.heroku.com/articles/dynos#ephemeral-filesystem

**Comment by devsoftzz** [comment]
Ohh...It may cause issues. Better to rely on paid hosts for production projects :p
